az-core/src/main/java/azkaban/Constants.java

```diff
@@ -684,15 +684,23 @@ public static class ContainerizedDispatchManagerProperties {
         "flow.container.";
     public static final String KUBERNETES_FLOW_CONTAINER_NAME =
         KUBERNETES_FLOW_CONTAINER_PREFIX + ".name";
+    public static final String KUBERNETES_FLOW_CONTAINER_CPU_RECOMMENDATION_MULTIPLIER =
+        KUBERNETES_FLOW_CONTAINER_PREFIX + "cpu.recommendation.multiplier";
     public static final String KUBERNETES_FLOW_CONTAINER_CPU_LIMIT_MULTIPLIER =
         KUBERNETES_FLOW_CONTAINER_PREFIX + "cpu.limit.multiplier";
+    public static final String KUBERNETES_FLOW_CONTAINER_MIN_ALLOWED_CPU =
+        KUBERNETES_FLOW_CONTAINER_PREFIX + "min.allowed.cpu";
```

Comment:
It exists for ramping purpose.

Provide min allowed cpu for ramping up VPA in case if unfortunately VPA gives incorrect extremely low CPU recommendation. This config can be removed in the future once VPA feature is stable.